CRITICAL - 2024-02-23 00:03:22 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(77): App\Models\UsuariosModel->editarUsuario('4', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:03:31 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(77): App\Models\UsuariosModel->editarUsuario(null, [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:12:42 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(81): App\Models\UsuariosModel->editarUsuario('4', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:12:52 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(81): App\Models\UsuariosModel->editarUsuario('11', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:15:29 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('4', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:16:08 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('11', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:16:22 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('32', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:17:16 --> ErrorException: print_r() expects parameter 2 to be bool, array given
in APPPATH\Models\UsuariosModel.php on line 29.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r() expects parameter 2 to be bool, array given', 'C:\\xampp2\\htdocs\\newVersion\\app\\Models\\UsuariosModel.php', 29, [...])
 2 APPPATH\Models\UsuariosModel.php(29): print_r('32', [...])
 3 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('32', [...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:18:23 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(31): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('32', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:21:37 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(31): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('32', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:22:56 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(33): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('32', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:23:02 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(33): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('32', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 00:23:32 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1778.
 1 SYSTEMPATH\BaseModel.php(1778): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(956): CodeIgniter\BaseModel->transformDataToArray(null, 'update')
 3 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], null)
 4 APPPATH\Models\UsuariosModel.php(33): CodeIgniter\Model->update([...])
 5 APPPATH\Controllers\UsuariosController.php(74): App\Models\UsuariosModel->editarUsuario('32', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->editarUsuario()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:57:14 --> Error: Call to a member function validarLogin() on null
in APPPATH\Controllers\UsuariosController.php on line 28.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:57:48 --> Error: Call to a member function validarLogin() on null
in APPPATH\Controllers\UsuariosController.php on line 28.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 01:58:04 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 01:58:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:58:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:58:04 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 01:58:23 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 01:58:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:58:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:58:23 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 01:59:15 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 01:59:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:15 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:15 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 01:59:22 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 01:59:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:22 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 01:59:43 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 01:59:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:43 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 01:59:59 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 01:59:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 01:59:59 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:00:14 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 02:00:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:00:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:00:14 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:00:23 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 02:00:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:00:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:00:23 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:00:52 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(NULL, NULL)
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 02:00:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:00:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:00:52 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` IS NULL
 LIMIT 1', [], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(28): App\Models\UsuariosModel->validarLogin(null, null)
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:01:01 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 02:01:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:01:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:01:01 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:01:31 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 02:01:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:01:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:01:31 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(29): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:02:41 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 02:02:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:02:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:02:41 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:02:49 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
#7 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#10 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-02-23 02:02:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
 5 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:02:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
 7 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:02:49 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:03:17 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(30): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
#7 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-02-23 02:03:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:03:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:03:17 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\UsuariosModel.php(30): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-02-23 02:04:03 --> mysqli_sql_exception: Unknown column 'nombre' in 'where clause' in C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp2\htdocs\newVersion\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp2\htdocs\newVersion\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp2\htdocs\newVersion\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\newVersion\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\newVersion\app\Models\UsuariosModel.php(32): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp2\htdocs\newVersion\app\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
#7 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
#8 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
#9 C:\xampp2\htdocs\newVersion\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\newVersion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-02-23 02:04:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:', [...], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\UsuariosModel.php(32): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:04:03 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:', [...], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\UsuariosModel.php(32): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:04:03 --> [Caused by] mysqli_sql_exception: Unknown column 'nombre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `usuarios`
WHERE `nombre` = \'cude\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `nombre` = :nombre:', [...], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\UsuariosModel.php(32): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:05:41 --> Error: Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
in APPPATH\Models\UsuariosModel.php on line 32.
 1 APPPATH\Controllers\UsuariosController.php(30): App\Models\UsuariosModel->validarLogin('cude', '543242')
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:09:56 --> ParseError: syntax error, unexpected end of file, expecting function (T_FUNCTION) or const (T_CONST)
in APPPATH\Models\UsuariosModel.php on line 69.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp2\\htdocs\\newVersion\\app\\Models\\UsuariosModel.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\UsuariosModel')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\UsuariosModel')
 4 APPPATH\Controllers\UsuariosController.php(13): spl_autoload_call('App\\Models\\UsuariosModel')
 5 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\UsuariosController->__construct()
 6 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:10:08 --> ParseError: syntax error, unexpected end of file, expecting function (T_FUNCTION) or const (T_CONST)
in APPPATH\Models\UsuariosModel.php on line 69.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp2\\htdocs\\newVersion\\app\\Models\\UsuariosModel.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\UsuariosModel')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\UsuariosModel')
 4 APPPATH\Controllers\UsuariosController.php(13): spl_autoload_call('App\\Models\\UsuariosModel')
 5 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\UsuariosController->__construct()
 6 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:10:18 --> ParseError: syntax error, unexpected end of file, expecting function (T_FUNCTION) or const (T_CONST)
in APPPATH\Models\UsuariosModel.php on line 59.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp2\\htdocs\\newVersion\\app\\Models\\UsuariosModel.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\UsuariosModel')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\UsuariosModel')
 4 APPPATH\Controllers\UsuariosController.php(13): spl_autoload_call('App\\Models\\UsuariosModel')
 5 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\UsuariosController->__construct()
 6 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:20:13 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 970.
 1 SYSTEMPATH\BaseModel.php(970): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\UsuariosModel.php(57): CodeIgniter\Model->update()
 4 APPPATH\Controllers\UsuariosController.php(99): App\Models\UsuariosModel->eliminarUsuario('11')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->eliminarUsuario()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:20:18 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 970.
 1 SYSTEMPATH\BaseModel.php(970): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\UsuariosModel.php(57): CodeIgniter\Model->update()
 4 APPPATH\Controllers\UsuariosController.php(99): App\Models\UsuariosModel->eliminarUsuario('11')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->eliminarUsuario()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:20:31 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 970.
 1 SYSTEMPATH\BaseModel.php(970): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\UsuariosModel.php(57): CodeIgniter\Model->update()
 4 APPPATH\Controllers\UsuariosController.php(99): App\Models\UsuariosModel->eliminarUsuario('11')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->eliminarUsuario()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-23 02:20:59 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 970.
 1 SYSTEMPATH\BaseModel.php(970): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\UsuariosModel.php(57): CodeIgniter\Model->update()
 4 APPPATH\Controllers\UsuariosController.php(99): App\Models\UsuariosModel->eliminarUsuario('11')
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UsuariosController->eliminarUsuario()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UsuariosController))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
